docs/cli/plugin_implementation_guide.md

```diff
@@ -39,12 +41,55 @@ filesystem repository in the creation and testing of their feature.
 
 #### Adding Admin Repository
 
+NOTE: This is not applicable if context-aware plugin discovery is enabled within Tanzu CLI.
```

Comment:
As above: Perhaps "context-aware plugin discovery" should be a link to one of the documentation sections from one of the files above?